Models/Aircraft.cs

```diff
@@ -5,7 +5,7 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-
+//1
 namespace FlightsCode.Models
 {
     public class Aircraft
```

Comment:
Visos properties, kurias tu priskiri per konstruktoriu public. Jos gali buti uzsetinamos is isores. Properties settinimas per konstruktoriu butu logiska, jei jos butu privacios.